### DISCUSSION TITLE
How to avoid erb parse <%%> in *.md code block?

### DISCUSSION THREAD

**Discussion by qichunren** [bridgetownrb/bridgetown#530] [Q&A]
I create a site with default erb template engine, there are code block in src/_posts *.md file, that contains <% %> in code block. It seems that bridgetown parse <%%> as normal erb ruby code.

This make me failed to post erb code sample in my post content.

**Comment by ayushn21** [comment]
You can ringfence code blocks using ```. So for example:

`` ``` ``
`<%= "This will not be processed as ERB" %>`
`` ``` ``

**Reply by qichunren** [reply]
Thanks for your reply. That's exactly what I originally did.

**Reply by qichunren** [reply]
I test your code in a new created bridgetown project: `bridgetown new bt_test -c stimulus,tailwindcss -t erb -e webpack`

Then add following code block in default first post `welcome-to-bridgetown.md`

<pre>
```
<%= "This will not be processed as ERB #{1+1}" %>
```
</pre>

The expected result in final html render should be:
```
<%= "This will not be processed as ERB #{1+1}" %>
```
But in fact it's:
<img width="423" alt="image" src="https://user-images.githubusercontent.com/39784/162218447-4afd10da-da20-4478-87ae-061143d063cf.png">


That is to say, bridgetown parsed ERB code in code block in a markdown post.

**Reply by ayushn21** [reply]
Try writing the ERB tag this way:

`<%%= "This will not be processed as ERB #{1+1}" %>`

That should hopefully do the trick!
